dojo/dojo-init

```diff
@@ -36,9 +36,11 @@ define () {
 DEFAULT_DOJO_HOST=localhost.pwn.college
 
 define DOJO_HOST "${DEFAULT_DOJO_HOST}"
+define WORKSPACE_HOST workspace."${DEFAULT_DOJO_HOST}"
```

Comment:
This is also super critical for a production migration, it ultimately gets used for ssl cert grabbing.